aequilibrae/paths/link.py

```diff
@@ -0,0 +1,56 @@
+# temporary containers for bush-based assignment integration
+class Node:
+    def __init__(self, node_id=0, node_index=0):
+        self.node_id = node_id
+        self.node_index = node_index
+
+
+class Link:
+    def __init__(self, node_id_from, node_id_to, link_id, t0, capacity, alfa, beta, direction=1, t0_ba=None, c_ba=None):
+        self.link_id = link_id
+        self.node_id_from = node_id_from
+        self.node_id_to = node_id_to
+        self.link_index = None
+        self.t0 = t0
+        self.capacity = capacity
+        self.alfa = alfa
+        self.beta = beta
+        self.direction = direction
+        self.t0_ba = t0_ba
+        self.capacity_ba = c_ba
+
+    # import math
```

Comment:
Shouldn't we delete all this commented stuff?

Reply:
Agreed, in fact I'd like to remove the link class altogether